src/main/scala/mesosphere/marathon/upgrade/DeploymentManager.scala

```diff
@@ -50,6 +50,15 @@ class DeploymentManager(
         case _ => origSender ! ConflictingDeploymentsCanceled(plan.id)
       }
 
+    case CancelAllDeployments =>
+      val origSender = sender()
+      val res = runningDeployments.keys map { id =>
+        stopActor(runningDeployments(id).ref, new Exception("Removing all running deployments"))
```

Comment:
Could we use a more specific exception here?  Feel free to define a new one if you don't find any of the existing exception types acceptable. See https://github.com/mesosphere/marathon/blob/ffc7ca3cf4789c954b426cc45927767e7b7bf3be/src/main/scala/mesosphere/marathon/Exception.scala#L28.